search-commons/src/main/java/no/unit/nva/search2/resource/ResourceClient.java

```diff
@@ -27,7 +27,7 @@ public class ResourceClient extends OpenSearchClient<SwsResponse, ResourceSearch
 
     private final UserSettingsClient userSettingsClient;
     @SuppressWarnings("PMD.DoNotUseThreads")
-    private static final ExecutorService executorService = Executors.newFixedThreadPool(5);
+    private static final ExecutorService executorService = Executors.newFixedThreadPool(3);
```

Comment:
Is the assignee the reason for decreasing the thread pool? In that case it is tempting to force the frontend to make two queries on their own and merge them. As this I would reckon causes all searches to a bit slower.

Reply:
To be honest, I am a bit confused as of why it is marked as a change, as I didn't change it now.

This code will never issue more than two async requests, so three should be enough...